source/d3d8to9_device.cpp

```diff
@@ -851,14 +851,17 @@ HRESULT STDMETHODCALLTYPE Direct3DDevice8::GetTexture(DWORD Stage, IDirect3DBase
 		case D3DRTYPE_TEXTURE:
 			BaseTextureInterface->QueryInterface(IID_PPV_ARGS(&TextureInterface));
 			*ppTexture = ProxyAddressLookupTable->FindAddress<Direct3DTexture8>(TextureInterface);
+			TextureInterface->Release();
```

Comment:
I'd maybe just call `BaseTextureInterface->Release()` in all case blocks (including the `default` actually, for completeness, not that it would ever be reached). Functionally there is no difference, but logically the code is using the queried interface for the proxy lookup (which is what is returned), so looks cleaner to keep the reference on those particular variables I think.